project/src/main/webapp/sps/data/Event.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Date;
+import java.util.Random; 
+import java.util.Set;
+
+/** Class containing Event object 
+Setters for variables that user can change about event
+*/
+public class Event {
+  private final int eventId;
+  private String name;
+  private String description;
+  private Set<String> labels;
+  private String location;
+  private Date date;
+  private Set<VolunteeringOpportunity> opportunities;
+  private Set<User> attendees;
+  private User host;
+
+  Random rand = new Random(); 
+
+  /** TO DO (MVP): Add Event to Event db*/
+  public Event(String name, String description, Set<String> labels, String location, Date date, 
+  Set<VolunteeringOpportunity> opportunities, Set<User> attendees, User host) {
+    this.eventId = rand.nextInt(1000000); //this will later be a unique number based on the database id
```

Comment:
Why do you need a random number here?

Reply:
Technically the number wont be random, it should be a unique number as the event ID which will come from the unique hash of the  event in the database -- i just put a random large number for now to prevent ID collisions
This is all hardcoded for the prototype which will change next week -- however I can also change this to make it a guaranteed unique int
(We wont create more than 10 events for the prototype)

Reply:
resolved